### DISCUSSION TITLE
Console breakout?

### DISCUSSION THREAD

**Discussion by cryptoknight** [cloud-hypervisor/cloud-hypervisor#3978] [Q&A] [answered]
I just booted my first VM with Cloud but forgot to configure a user first so am stuck at login prompt. Is there a keyboard shortcut to exit the console or shutdown the VM without having to manually kill the cloud-hypervisor process?

**Comment by sboeuf** [comment]
Not sure how to do that, but that's not related to Cloud Hypervisor, it depends on the OS/image you're using.

**Comment by rbradford** [comment, marked as answer]
> I just booted my first VM with Cloud but forgot to configure a user first so am stuck at login prompt. Is there a keyboard shortcut to exit the console or shutdown the VM without having to manually kill the cloud-hypervisor process?

If you set an API socket then you can use `ch-remote power-button`
